### PR TITLE
Add reactive `image` and `title` to `UIBarItem`

### PR DESCRIPTION
**Pull request by eimantas**

### CODE REVIEW

File: ReactiveCocoaTests/UIKit/UIBarButtonItemSpec.swift [hunk 1]
```diff
@@ -35,5 +35,30 @@ class UIBarButtonItemSpec: QuickSpec {
 			observer.send(value: true)
 			expect(barButtonItem.isEnabled) == true
 		}
+
+		it("should accept changes from bindings to its title") {
+			let (pipeSignal, observer) = Signal<String?, NoError>.pipe()
+			barButtonItem.reactive.title <~ SignalProducer(signal: pipeSignal)
+
+			observer.send(value: "title")
+			expect(barButtonItem.title).to(equal("title"))
```
Comment (NachoSoto) [inline ReactiveCocoaTests/UIKit/UIBarButtonItemSpec.swift]: Can you use `==` instead?

File: ReactiveCocoaTests/UIKit/UIBarButtonItemSpec.swift [hunk 2]
```diff
@@ -35,5 +35,30 @@ class UIBarButtonItemSpec: QuickSpec {
...
+
+		it("should accept changes from bindings to its image") {
+			let (pipeSignal, observer) = Signal<UIImage?, NoError>.pipe()
+			barButtonItem.reactive.image <~ SignalProducer(signal: pipeSignal)
+
+			let image = UIImage()
+			expect(image).notTo(beNil())
+
+			observer.send(value: image)
+			expect(barButtonItem.image).to(equal(image))
```
Comment (NachoSoto) [inline ReactiveCocoaTests/UIKit/UIBarButtonItemSpec.swift]: Ditto

Comment (eimantas) [inline ReactiveCocoaTests/UIKit/UIBarButtonItemSpec.swift]: Should I use `==` for comparison with `nil` too?

**Comment by mdiep** [comment]
Thanks!

**Pull request merged**